contract/contracts/PixelDevsUkraineDonation.sol

```diff
@@ -35,8 +36,10 @@ contract PixelDevsUkraineDonation is
         uint256 indexed newValue
     );
     event ContractStateUpdated(bool indexed oldValue, bool indexed newValue);
+    event WithdrawWalletUpdated(address indexed oldValue, address indexed newValue);
 
-    constructor() ERC721("PixelDevsUkraineDonation", "PXLDEV-UKRAINE") {
+    constructor(address _withdrawWallet) ERC721("PixelDevsUkraineDonation", "PXLDEV-UKRAINE") {
```

Comment:
Also the constructor argument as well, when we are hard coding the wallet address for Ukraine DAO